cmd/sovereignnode/sovereignNodeRunner.go

```diff
@@ -1224,6 +1224,7 @@ func (snr *sovereignNodeRunner) CreateManagedStatusComponents(
 		IsInImportMode:       isInImportMode,
 		StatusCoreComponents: managedStatusCoreComponents,
 		CryptoComponents:     cryptoComponents,
+		IsSovereign:          true,
```

Comment:
Please also add it specifically to `false` in `CreateManagedStatusComponents` from `nodeRunner.go` for extra transperancy